components/src/core/InfoListItem/InfoListItem.tsx

```diff
@@ -251,18 +244,14 @@ const InfoListItemRender: React.ForwardRefRenderFunction<unknown, InfoListItemPr
 
     const getInfoListItemContent = (): JSX.Element => (
         <>
-            <StatusStripe
-                statusColor={statusColor}
-                className={combine('statusStripe')}
-                data-testid={'blui-status-stripe'}
-            />
-            {divider && <InfoListItemDivider divider={divider} className={combine('divider')} />}
+            <StatusStripe statusColor={statusColor} className={'statusStripe'} data-testid={'blui-status-stripe'} />
+            {divider && <InfoListItemDivider divider={divider} className={'divider'} />}
             {(icon || !hidePadding) && getIcon()}
             {leftComponent}
             <InfoListItemText
                 primary={title}
                 leftComponent={leftComponent}
-                className={combine('listItemText')}
+                className={generatedClasses.listItemText}
```

Comment:
Let's be consistent with using `generatedClasses.XXX` or `generatedClasses['XXX']` — the mix is confusing. Most of the other files are using the former.